analyzer-models/src/main/java/ffb/analyzer/models/espn/PlayerRanking.java

```diff
@@ -0,0 +1,82 @@
+package ffb.analyzer.models.espn;
+
+/**
+ * Entity representing a player's ranking.
+ */
+public class PlayerRanking extends EspnEntity<PlayerRanking> {
+
+    public enum RankType {
+        STANDARD,
+        PPR
+    }
+
+    private int auctionValue;
+    private int rank;
+    private int rankSourceId;
+    private RankType rankType;
+    private int slotId;
+    private int id;
+
+    /**
+     * Default constructor.
+     */
+    public PlayerRanking() {
```

Comment:
Why do we need the default constructor? We're handling the deserialization ourselves.